packages/client/hooks/useUnusedRecords.ts

```diff
@@ -3,19 +3,21 @@ import {Threshold} from '../types/constEnums'
 
 const useUnusedRecords = (
   serviceTaskIds: string[],
-  usedServiceTaskIds: Set<string>
+  usedServiceTaskIds: Set<string>,
+  thresh?: number
 ): [string[], boolean | null] => {
   return useMemo(() => {
     const unusedServiceTaskIds = [] as string[]
     serviceTaskIds.forEach((servieTaskId) => {
       if (!usedServiceTaskIds.has(servieTaskId)) unusedServiceTaskIds.push(servieTaskId)
     })
+    const maxStories = thresh ? thresh : Threshold.MAX_POKER_STORIES
```

Comment:
+1 you could make the `tresh` param mandatory and provide `Threshold.MAX_POKER_STORIES` as default, then this line could be removed